src/pages/ExamsPage/index.test.jsx

```diff
@@ -18,31 +20,48 @@ describe('ExamsPage', () => {
     ],
     isLoading: false,
   };
-  describe('snapshots', () => {
-    test('loaded', () => {
-      hooks.useExamsData.mockReturnValue(defaultExamsData);
-      expect(render(<ExamsPage courseId="test_course" />)).toMatchSnapshot();
+  const defaultAttemptsData = {
+    attemptsList: [{
+      exam_name: 'Exam 1',
+      username: 'username',
+      time_limit: 60,
+      exam_type: 'Timed',
+      started_at: '2023-04-05T19:27:16.000000Z',
+      completed_at: '2023-04-05T19:27:17.000000Z',
+      status: 'completed',
+    }],
+  };
+  describe('loading tests', () => {
+    beforeEach = (() => { // eslint-disable-line no-global-assign
+      hooks.useExamAttemptsData.mockReturnValue(defaultAttemptsData);
     });
-  });
-  describe('loading message', () => {
-    it('should render while fetching data', () => {
-      hooks.useExamsData.mockReturnValue({
-        ...defaultExamsData,
-        isLoading: true,
+    describe('snapshots', () => {
```

Comment:
any reason for moving these around? The snapshot test is covering everything (loading message, tab navigation, data grid) but moving it here implies it's just a sub test of the loading message feature.

Reply:
My idea was that I could call beforeEach before both tests to avoid calling `hooks.useExamAttemptsData.mockReturnValue(defaultAttemptsData);` the same way in both tests, but looking at it know it kind of makes it more messy (plus it's only two tests) so I'm gonna put it back.

Reply:
If you want to avoid repeating the setup I think you can just do a beforeAll under the describe('ExamsPage' block since that mock never changes in these tests